docs/advanced-features/react-18/switchable-runtime.md

```diff
@@ -8,19 +8,6 @@ Next.js has two **server runtimes** where you can render parts of your applicati
 
 By default, Next.js uses the Node.js runtime. [Middleware](https://nextjs.org/docs/advanced-features/middleware) and [Edge API Routes](https://nextjs.org/docs/api-routes/edge-api-routes) use the Edge runtime.
 
-## Global Runtime Option
-
-To configure the runtime for your whole application, you can set the experimental option `runtime` in your `next.config.js` file:
-
-```js
-// next.config.js
-module.exports = {
```

Comment:
if this has been moving out because isn't experimental anymore, could be still nice to keep this example to show how to setup global configuration 🙂